client/src/pages/AboutPage/components/NetworkingSection/NetworkingSection.css

```
@@ -0,0 +1,21 @@
+.networking-section {
+	margin-bottom: 30px;
+}
```

As we discussed/agreed on the Design Philsophy the last section of the page doesn't need a margin-bottom.

```suggestion
.networking-section {
		/* placeholder */
}
```

---

Yes, we need not to forget to add margin to main:)